### PR TITLE
Datetime bug with dealing with undefined values

### PR DESCRIPTION
**Pull request by Synicix**
- This is bug I found with the IBL tables. It has to do with the rewrite so I didn't push a change log.
- Depends #102 (Sorry was working on this before I found this)
- Bug fixed with issue of date time input block erroring out when undefined is passed as current value. This is an issue that there isn't an agree upon value for date_time value yet.

### CODE REVIEW

**Comment by Synicix** [comment]
Can we cram this in the Beta.1? @guzman-raphael

**Comment by Synicix** [comment]
I patch it temporarly for now, but yeah we really need to consildate on undefined for current value for tableAttribute

**Comment by Synicix** [comment]
Update from dev now

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@Synicix Now that we have released in `BETA`, we will need to add a changelog entry for this bug fix. It is unclear exactly the use case where `currentValue` comes in as an array.

Please update the following:
- Changelog entry
- PR description of the actual bug
- Screenshot of the issue/fix
- Update GH Milestone/Project assignment

**Comment by Synicix** [comment]
I was hopping to shove it in beta cause it was broken due to the overall to allow copy over to work the other day.

**Review by mahos — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 - CSS for making the primary keys look disabled/readonly for update mode PR #105
 
 ### Fixed
-- Fix bug of delete with datetime in primarykey crashing PR #105
+- Fixed bug of delete with datetime in primarykey crashing PR #105
+- Fixed issue website crashing when opening generate an input html block for datetime (#104) PR #106
```
Comment (guzman-raphael) [inline CHANGELOG.md:12]:
```suggestion
- Fixed issue where website crashes when opening a filter card for datetime. (#104) PR #106
```

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@Synicix Also update the PR description. It was not clear what this PR fixes until I checked the linked issue.

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**